.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no -- commitlint --edit "$1"
```

Comment:
Should we use the `scripts` field of `package.json` instead? That way, we can easily invoke the check with `npm run commitlint`. See my comments in file `package.json`.

```suggestion
npm run commitlint -- --edit "$1"
```